posthog/session_recordings/test/test_session_recording_helpers.py

```diff
@@ -225,6 +231,7 @@ def test_received_snapshot_source_is_respected_for_first_event(raw_snapshot_even
                 "$snapshot_data": {"type": 3, "timestamp": MILLISECOND_TIMESTAMP},
                 "distinct_id": "abc123",
                 "$snapshot_source": "mobile",
+                "$lib": "posthog-ios/1.0.0",
```

Comment:
I am a bit confused with this test, the user agent has the lib/version format, the $lib prop is just 'posthog-{lib}'.